{{cookiecutter.project_slug}}/src/redux/mainReducer.js

```diff
@@ -3,17 +3,17 @@ import { combineReducers } from "redux";
 /**
  * You can import more reducers here
  */
-{% if cookiecutter.has_email_auth_blueprint == "y" %}import { EmailAuthReducer } from "../features/EmailAuth/redux/reducers";{% endif %}
-{% if cookiecutter.has_calendar_blueprint == "y" %}import { CalendarReducer } from "../features/Calendar/redux/reducers";{% endif %}
+
+
 //@BlueprintReduxImportInsertion
 
 export const combinedReducers = combineReducers({
   blank: (state, action) => {
     if (state == null) state = [];
     return state;
   },
-  {% if cookiecutter.has_email_auth_blueprint == "y" %}EmailAuth: EmailAuthReducer,{% endif %}
-  {% if cookiecutter.has_calendar_blueprint == "y" %}Calendar: CalendarReducer,{% endif %}
```

Comment:
Before we merge this PR this needs to be implemented in the `install.sh` of the `EmailAuth` and `Calendar` screens repositories.